src/main/bicep/mainTemplate.bicep

```diff
@@ -322,6 +322,6 @@ module singleServerEndPid './modules/_pids/_empty.bicep' = {
   ]
 }
 
-output adminSecuredConsole string = uri(format('https://{0}:9043/', const_newVNet ? publicIPAddress.properties.dnsSettings.fqdn : reference(name_networkInterfaceNoPubIp).ipConfigurations[0].properties.privateIPAddress), 'ibm/console')
+output adminSecuredConsole string = uri(format('https://{0}:9043/', const_newVNet ? publicIPAddress.properties.dnsSettings.fqdn : reference(name_networkInterfaceNoPubIp).ipConfigurations[0].properties.privateIPAddress), 'ibm/console/logon.jsp')
```

Comment:
Always good to save a round trip. On the other hand, if the console stops using `logon.jsp` some day, this link won't work. However, there is little risk of that because tWAS is not actively developed.